src/uu/ls/src/ls.rs

```diff
@@ -611,6 +639,10 @@ pub fn uumain(args: impl uucore::Args) -> UResult<()> {
         .map(|v| v.map(Path::new).collect())
         .unwrap_or_else(|| vec![Path::new(".")]);
 
+    if config.context && !config.selinux_supported {
+        show_warning!("--context (-Z) works only on an SELinux-enabled kernel",);
+    }
+
```

Comment:
GNU ls doesn't seem to do this, so perhaps there should only be a warning if the flag is used on a `uu_ls` build without the feature enabled?

Reply:
Yes, good point. I'll do that.